Twits/Controllers/TwitController.cs

```diff
@@ -0,0 +1,214 @@
+                    WHERE Id = @id
+                ";
+                SqlCommand command = new SqlCommand(sql, connection);
+                command.Parameters.AddWithValue("@id", id);
+                SqlDataReader reader = command.ExecuteReader();
+                Twit twit = new Twit();
+
+                while (reader.Read())
+                {
+                    twit.Id = reader.GetInt32(0);
+                    twit.Text = reader.GetString(1);
+                    twit.CreatedOn = reader.GetDateTimeOffset(2);
+                }
+
+                return View(twit);
+            }
+        }
+
+        //// POST: Twit/Edit/5
+        [HttpPost]
+        public ActionResult Edit(int id, FormCollection collection)
+        {
+            Twit twit = new Twit();
+            twit.Id = id;
+            twit.Text = collection.Get("Text");
+            try
+            {
+                string connectionString = ConfigurationManager.ConnectionStrings["TwitDatabase"].ConnectionString;
+
+                using (SqlConnection connection = new SqlConnection(connectionString))
+                {
+                    connection.Open();
+                    string sql = @"
+                        UPDATE Twits
+                        SET Text = @Text
+                        WHERE Id = @Id
+                    ";
+                    SqlCommand command = new SqlCommand(sql, connection);
+                    command.Parameters.AddWithValue("@Text", twit.Text);
+                    command.Parameters.AddWithValue("@Id", id);
+                    command.ExecuteNonQuery();
+                }
+
+                return RedirectToAction("Index");
+            }
+            catch
+            {
+                return View(twit);
+            }
+        }
+
+        //// GET: Twit/Delete/5
+        //public ActionResult Delete(int id)
+        //{
+        //    return View();
+        //}
+
+        //// POST: Twit/Delete/5
+        [HttpPost]
+        public ActionResult Delete(int id, FormCollection collection)
+        {
+            Twit twit = new Twit();
+            //twit.Text = collection.Get("Text");
+            //twit.CreatedOn = DateTimeOffset.Now;
+            try
+            {
+                string connectionString = ConfigurationManager.ConnectionStrings["TwitDatabase"].ConnectionString;
+
+                using (SqlConnection connection = new SqlConnection(connectionString))
+                {
+                    connection.Open();
+                    string sql = @"
+                        Delete FROM Twits WHERE
+                        Id = @Id
+                    ";
+                    SqlCommand command = new SqlCommand(sql, connection);
+                    command.Parameters.AddWithValue("@Id", id);
+                    command.ExecuteNonQuery();
+                }
+
+                return RedirectToAction("Index");
+            }
+            catch
+            {
+                return View(twit);
```

Comment:
You don't have a view associated with this action method, so I'd just add another call to `RedirectToAction` here.